book/doctrine.rst

```diff
@@ -386,9 +391,10 @@ Generating Getters and Setters
 
 Even though Doctrine now knows how to persist a ``Product`` object to the
 database, the class itself isn't really useful yet. Since ``Product`` is just
-a regular PHP class, you need to create getter and setter methods (e.g. ``getName()``,
-``setName()``) in order to access its properties (since the properties are
-``protected``). Fortunately, Doctrine can do this for you by running:
+a regular PHP class with ``protected`` properties, you need to create ``public``
+getter and setter methods (e.g. ``getName()``, ``setName($name)``) in order
+to access its properties. Fortunately, the following command can generate
+these boilerplate methods automatically:
```

Comment:
I'd suggest maintaining the discussion of `doctrine:generate:entities`.  While its auto-generation of getters/setters is admittedly redundant with functionality provided by modern IDEs, it offers some capabilities beyond the realm of IDEs.  (I'm referring here to bullet points 2 and 3 under the "More about doctrine:generate:entities" subsection).   Also, I haven't seen any documentation on Doctrine's site discouraging the usage of this command.